opengrok-web/src/main/webapp/default/style.css

```diff
@@ -1249,6 +1236,34 @@ h6.message-group-caption {
     line-height: 1.3;
 }
 
+.important-note {
+    text-align: center;
+    display: inline-block;
+    max-height: 100%;
+}
+
+.note-success {
+    background-color: #3c763d;
+}
+
+.note-info {
+    background-color: #31708f;
+}
+
+.note-warning {
+    background-color: #8a6d3b;
+}
+
+.note-error {
+    background-color: #a94442;
+}
```

Comment:
a question here is if you want to restrict these classes to be used only for `important-note` or just without any restriction (one can use it later for completely unrelated styling with the given colour).

Reply:
Is there a way how to limit the `note-*` styles for both `important-note` and `.important-note.important-note-rounded`. The thing is, `.important-note` is used both with and without `important-note-rounded`.

Reply:
Initially, I wanted to unite the `note-*` styles with `.message-group-item.*` styles so the color definitions are defined just once however the latter use `color` instead of `backgound-color`.

Reply:
By elegant I mean something like this:
```CSS
.important-note {
    text-align: center;
    display: inline-block;
    max-height: 100%;
}

.important-note.note-success {
    background-color: #3c763d;
}

... // more .note styles
```
and then it would be nice if I can say something like this:
```CSS
.important-note.note-*.important-note-rounded {
    border-radius: 50%;
    max-width: 100%;
    min-width: 20px;
}
```
however I am not sure this is possible in CSS.

Reply:
Follow-up on our Slack discussion: it turns out that I misread the CSS selectors guide - the `.class1.class2` selector is not strictly hierarchical so there can be style that can only be used with `important-note` while being also usable with both `important-note.note` and `important-note-rounded`.